core/src/main/java/com/scalar/db/storage/jdbc/query/SimpleSelectQuery.java

```diff
@@ -89,6 +103,51 @@ private String conditionSqlString() {
     return String.join(" AND ", conditions);
   }
 
+  private String relationalConditionSqlString() {
+    if (conjunctions.isEmpty()) {
+      return "";
+    }
+
+    List<String> conjunctionList = new ArrayList<>();
+    conjunctions.forEach(
+        conjunction -> {
+          List<String> conditions = new ArrayList<>();
+          conjunction
+              .getConditions()
+              .forEach(
+                  condition ->
+                      conditions.add(
+                          rdbEngine.enclose(condition.getColumn().getName())
+                              + convert(condition.getOperator())));
+          conjunctionList.add(String.join(" AND ", conditions));
+        });
+
+    return " WHERE " + String.join(" OR ", conjunctionList);
+  }
+
+  private String convert(ConditionalExpression.Operator operator) {
+    switch (operator) {
+      case EQ:
+        return "=?";
+      case NE:
+        return "!=?";
+      case GT:
+        return ">?";
+      case GTE:
+        return ">=?";
+      case LT:
+        return "<?";
+      case LTE:
+        return "<=?";
+      case IS_NULL:
+        return " IS NULL";
```

Comment:
[minor] The above condition strings don't have leading space, but these 2 ones have. How about making the all strings consistent if possible?

Reply:
Thank you for the suggestion. For "is null" and "is not null", we cannot remove the space since they must be separated from the column name. In contrast, we end up creating inconsistency with other places like `conditionSqlString()` if we add spaces for other conditions. So, can we keep it as is?

Reply:
Sounds reasonable! I think we can go with it as is.